clients/py/sapphirepy/sapphire.py

```diff
@@ -0,0 +1,135 @@
+from typing import Any, Callable, cast, TypedDict
+from binascii import unhexlify, hexlify
+
+from web3 import Web3
+from web3.types import RPCEndpoint, RPCResponse, TxParams
+from eth_typing import HexStr
+
+from .envelope import TransactionCipher
+
+# Should transactions which deploy contracts be encrypted?
```

Comment:
> "When it comes to privacy and accountability, people always demand the former for themselves and the latter for everyone else." - David Brin

Reply:
By default transactions which deploy code are unencrypted, but there is a manual toggle to turn on deployment transactions.